platform/src/lib.rs

```diff
@@ -16,6 +16,14 @@ pub mod printer;
 pub const MAX_CHUNK_SIZE: usize = 2048;
 pub const MAX_SN_SIZE: usize = 20;
 
+pub enum SignerIdentifier<'a> {
+    IssuerAndSerialNumber {
+        issuer_name: &'a [u8],
+        serial_number: &'a [u8],
+    },
+    SubjectKeyIdentifier(&'a [u8]),
+}
+
```

Comment:
If you make all the values here ArrayVecs, you can simplify the `get_signer_identifier` function signature a bit. Either way, I think the values will get copied into fixed-size temporary stack buffers.

Reply:
Done